context.md

```diff
@@ -0,0 +1,26 @@
+# **Context**
+
+Context is the object that a function is attached to.
+
+
+When a Javascript function is invoked, a context is set.  The context will be the same value of the object that owns the running code at that moment.  The value can be referenced with `this`.  Basically, `this` is a substitute for the object in question.
+
```

Comment:
That's good, but I would make one adjustment. It's really the object that has a context. 

But actually your first sentence touches on something interesting with functions. As we'll learn in the Object Oriented programming class, we can use functions as **constructors** with the `new` keyword. 

Don't worry about covering that here though!

Reply:
Good work!